projects/components/src/table/table.component.ts

```diff
@@ -676,7 +680,10 @@ export class TableComponent
       )
     );
 
-    this.tableCsvDownloaderService.executeDownload(downloadConfig$, this.id);
+    this.tableCsvDownloaderService.executeDownload(
```

Comment:
Should we add the confirmation logic as well?

Reply:
we should. will add in a separate follow up.